src/value_converter.rs

```diff
@@ -849,6 +928,11 @@ pub fn py_to_rust(parameter: &pyo3::Bound<'_, PyAny>) -> RustPSQLDriverPyResult<
             return Ok(PythonDTO::PyDateTime(pydatetime_no_tz));
         }
 
+        let timestamp_tz = extract_datetime_from_python_object_attrs(parameter);
```

Comment:
You specified many good Error descriptions, but here you ignore any Error from `extract_datetime_from_python_object_attrs`, so propagating errors is a good idea.

Reply:
I handle this case like here:
`let timestamp_tz = parameter.extract::<DateTime<FixedOffset>>();
        if let Ok(pydatetime_tz) = timestamp_tz {
            return Ok(PythonDTO::PyDateTimeTz(pydatetime_tz));
        }`

thus, this is no ignore any Error